docs/tutorials/RemovingFields.adoc

```diff
@@ -0,0 +1,97 @@
+= Tutorial - Removing a field from an entity
+:site-section: DeveloperGuide
+:toc: macro
+:toc-title:
+:imagesDir: ../images/remove
+:stylesDir: ../stylesheets
+:xrefstyle: full
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:warning-caption: :warning:
+:source-highlighter: highlightjs
+endif::[]
+
+[quote, Charles Maturin, Motto to "Bertram"]
+'Tis well to be merry and wise,
+'Tis well to be honest and true;
+'Tis well to be off with the old love,
+Before you are on with the new.
+
+{empty} +
+
+== Introduction
+When working on Address Book, you will most likely find that some features and fields that are no longer necessary.
+In scenarios like this, you can consider refactoring the existing `Person` model to suit your use case.
+
+In this tutorial, we'll do exactly just that and remove the `Address` field from `Person`.
+
+== Safely deleting `Address`
+
+Fortunately, the IntelliJ IDEA provides a robust refactoring tool that can identify _most_ usages.
+Let's try to use it as much as we can.
+
+=== Assisted refactoring
+We start by identifying ``Address``'s usages.
```

Comment:
Explain: What is the rationale behind this step? i.e., what's the connection between the `Address` field and this class?

Reply:
Added a bit to distinguish between the `address` member in `Person` and the `Address` class.